src/lib/validator.js

```diff
@@ -0,0 +1,8 @@
+const isValidNino = (value) => {
+  return value.match(
+    "" ||
+      /^(?!BG|GB|KN|NK|NT|TN|ZZ)[A-CEGHJ-PR-TW-Z][A-CEGHJ-NPR-TW-Z][0-9]{6}[ABCD]$/i
```

Comment:
This matches the format on HMRC's website  - https://www.gov.uk/hmrc-internal-manuals/national-insurance-manual/nim39110

Given they're big strings of letters, what do we think it more readable?
- `A-CEGHJ-PR-TW-Z`
- `ABCEGHJQLMNOPRSTWXZ`

Reply:
I think the big string way is more readable. I've also made this change.